### PR TITLE
Js dependencies

### PR DESCRIPTION
**Pull request by wadie**
Reopened a new PR to merge when your changes over at #241 are done. 👍

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -13,10 +13,12 @@
     <link rel="apple-touch-icon" sizes="72x72" href="/images/apple-72.png">
     <link rel="apple-touch-icon" sizes="114x114" href="/images/apple-114.png">
     <link rel="apple-touch-icon" sizes="144x144" href="/images/apple-144.png">
+    <link href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/default.min.css" rel="stylesheet" type="text/css">
```
Comment (marcoow) [inline _includes/head.html:16]: Can we load this *only* for the blog layout? It's not needed on any other page.

File: _includes/head.html [hunk 2]
```diff
@@ -13,10 +13,12 @@
...
+    <link href="/js/main.js">
```
Comment (marcoow) [inline _includes/head.html]: This should be

```html
<script src="/js/main.js"></script>
```

**Review by marcoow — CHANGES_REQUESTED** [review]
Is any of the gulp code going to run during the normal Jekyll build?

**Comment by ghislaineguerin** [comment]
@wadie I've changed the base to redesign as we no longer have the CSS-refactor branch

**Comment by marcoow** [comment]
this should not be needed anymore after #248

**Pull request closed without merging**